convert2rhel/grub.py

```diff
@@ -572,6 +581,31 @@ def post_ponr_set_efi_configuration():
         _log_critical_error(e.message)
 
 
+def update_grub_after_conversion():
+    """Update grub images and config after conversion."""
+    grub2_BIOS_config_file = (
+        GRUB2_BIOS_CONFIG_FILE if not is_efi() else os.path.join(RHEL_EFIDIR_CANONICAL_PATH, "grub.cfg")
+    )
+
+    # Generate the grub.cfg file
+    _, ret_code = utils.run_subprocess(["/usr/sbin/grub2-mkconfig", "-o", grub2_BIOS_config_file], print_output=False)
```

Comment:
Do you think that a simple if statement for not running this under RHEL 6 would be sufficient? Because we have the `grub-install` available under RHEL 6, but no `grub-mkconfig` or any other useful command at all...

The way I see it, we have the following options:: 

* Using an if-statment to no turn under RHEL 6 `if systeminfo.system_info.version.major <= 6: return` and prevent it from running the entire code if RHEL 6 is detected
* Ignoring the `grub2-mkconfig` using the same if, but actually let the code run for everything else and just adapt it to to run either grub2 or grub depending on the version 

What do you think it's best? @bocekm

Reply:
I've updated to ignore if it's RHEL6